### PR TITLE
Update perseus fonts to match wonderblocks

### PR DESCRIPTION
**Pull request by nixterrimus**
## Summary:
Updates perseus body font and passage font to match wonderblocks.

Screen Shot 2022-11-16 at 12 46 15 PM

Issue: https://khanacademy.atlassian.net/browse/LP-12340

## Test plan:
- Open storybooks
- Make sure everything is still readable

### CODE REVIEW

File: packages/perseus/src/styles/perseus-renderer.less [hunk 1]
```diff
@@ -107,7 +107,10 @@
     }
 
     div.paragraph {
-        .legacy-typography;
+        font-family: '"Noto Serif", serif';
```
Comment (jeremywiebe) [inline packages/perseus/src/styles/perseus-renderer.less:110]: Will this mean we need to ship new fonts with our mobile app?

**Comment by handeyeco** [comment]
The original ticket is to replace typography with WB and this PR is matching styles with WB. Since this is part 1, is the end goal still to replace typography with WB? Or was there a blocker and now we're shifting to a new solution?

The reason I ask is because if these are meant to be long-term changes I wonder if it'd be better to somehow share styles with WB rather than copy them so they don't get out of sync.

File: packages/perseus/src/styles/perseus-renderer.less (hunk 1 above)
Comment (nixterrimus) [reply packages/perseus/src/styles/perseus-renderer.less]: We should have these through wonderblocks. Do we need to add these to mobile? I would be surprised if any of perseus looked right without mostly having these baked in?

**Comment by nixterrimus** [comment]
@handeyeco I hear you. Wonderblocks encapsulates styles through React components and perseus uses global styles via a stylesheet. Yes this work can cause things to fall out of sync and in the longest of term I would like to use the Wonderblocks component for `<Body>` but in the short term I think this is better and the least risky to our shipping goals. So I'm not super worried about them falling out of sync. If we change Wonderblocks fonts I expect the whole org will be noisy about it and it'll be clear that we need to make changes.

**Comment by nixterrimus** [comment]
I had originally thought this would be a multistep process but I now feel content that this is a single PR for now.

**Comment by Myranae** [comment]
Question (non-blocking): Yay CSS updates! I'm curious, did you make these changes based on visually checking the style after changing the CSS? Are there design guidelines out there somewhere that should be followed when updating user facing content styles?

chore: Just wanted to point out that it looks like you might need to update your snapshots to get the checks to pass.

**Comment by nixterrimus** [comment]
@Myranae  yes! I'm following the wonderblocks styles. Here are some great resources to see it:

- https://wonderblocks.khanacademy.org
- https://wonder-blocks.netlify.app
- https://github.com/Khan/wonder-blocks

**Comment by nixterrimus** [comment]
I suspect this PR isn't the real culprit for breaking the building. Lemme see what I can do about that.

File: packages/perseus/src/styles/perseus-renderer.less (hunk 1 above)
Comment (jeremywiebe) [reply packages/perseus/src/styles/perseus-renderer.less]: 🤔  It looks like we have Noto for some non-en languages, but I don't see a base Noto font included anywhere in our codebase:

https://github.com/Khan/mobile/blob/develop/react-native/wonderblocks-rn/core/typography.js#L33

and 

```sh
$ find . -name "*Noto*"
./ios/Resources/NotoSansDevanagari-Light.ttf
./ios/Resources/NotoSansGeorgian-Light.ttf
./ios/Resources/NotoSansBengali-Bold.ttf
./ios/Resources/NotoSansArmenian-Bold.ttf
./ios/Resources/NotoSansArmenian-Thin.ttf
./ios/Resources/NotoSansArmenian-Regular.ttf
./ios/Resources/NotoSansDevanagari-Bold.ttf
./ios/Resources/NotoSansBengali-Black.ttf
./ios/Resources/NotoSansBengali-Regular.ttf
./ios/Resources/NotoSansArmenian-Black.ttf
./ios/Resources/NotoSansBengali-Light.ttf
./ios/Resources/NotoSansDevanagari-Regular.ttf
./ios/Resources/NotoSansGeorgian-Bold.ttf
./ios/Resources/NotoSansGeorgian-Regular.ttf
./ios/Resources/NotoSansGeorgian-Black.ttf
./ios/Resources/NotoSansDevanagari-Black.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Light.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Light.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Thin.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Black.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansArmenian-Black.ttf
./android/app/src/main/assets/fonts/NotoSansBengali-Light.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Bold.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Regular.ttf
./android/app/src/main/assets/fonts/NotoSansGeorgian-Black.ttf
./android/app/src/main/assets/fonts/NotoSansDevanagari-Black.ttf

$ 
```

Comment (nixterrimus) [reply packages/perseus/src/styles/perseus-renderer.less]: Here's a ticket to make sure this work gets done when we deploy on mobile: https://khanacademy.atlassian.net/browse/MOB-4782

File: packages/perseus/src/styles/perseus-renderer.less [hunk 2]
```diff
@@ -107,7 +107,10 @@
     }
 
     div.paragraph {
-        .legacy-typography;
+        font-family: "Noto Serif", serif;
```
Comment (jeremywiebe) [inline packages/perseus/src/styles/perseus-renderer.less:110]: **suggestion**: It _might_ be worth creating a variable for this font-family, but I don't feel super strongly about it.

**Review by jeremywiebe — APPROVED** [review]
Ok, I'm good with this. Glad to see more consistency with WB working its way into Perseus.

**Pull request merged**